instruments/cryomagnetics/cryomagnetics_lm510.py

```diff
@@ -0,0 +1,206 @@
+
+    measurement_timeout = 1
+
+    UNITS = {
+        "cm": pq.cm,
+        "in": pq.inch,
+        "%": pq.percent,
+        "percent": pq.percent
+    }
+
+    @property
+    def default_channel(self):
+        """
+        The Level meter has a default channel that is used to measure
+        different cryogen levels. If a channel is not specified for a
+        particular command, the command gets sent to the value of the
+        default channel instead
+
+        :return: The current default channel
+        :rtype: int
+        """
+        return int(self.query("CHAN?"))
+
+    @default_channel.setter
+    def default_channel(self, channel):
+        """
+        Set the default channel to the desired channel
+
+        :param int channel: The desired channel
+        :raises: :exc:`ValueError` if the channel to be set is not in the
+        set of allowed channels defined in ``CHANNELS``
+        """
+        if channel not in self.CHANNELS:
+            raise ValueError(
+                "Attempted to set channel to {0}. Channel must "
+                "be an integer of either 1 or 2".format(channel))
+        command = "CHAN {0}".format(channel)
+        self.query(command)
+
+    @property
+    def status_byte(self):
+        """
+        Get a byte array representing the instrument's status byte. The
+        status byte describes whether channel 1 or 2 are ready to make a
+        measurement. The structure of the status byte is described in
+        greater detail in the level meter's operating manual.
+
+        :return: The current value of the status byte
+        :rtype: bytes
+        """
+        byte_as_string = self.query("*STB?")
+        return byte_as_string.encode('ascii')
+
+    @property
+    def channel(self):
+        """
+
+        :return: The measurement channels for levels available to the level
+        meter.
+        """
+        return self.Channel(1, self), self.Channel(2, self)
+
+    def reset(self):
+        """
+        Bring the instrument to a safe, known state
+        """
+        self.query("*RST")
+
+    @contextmanager
+    def context_managed_measuring_lock(self):
+        """
+        Acquire the lock, and yield to the next function. Upon entry back
+        into the function, release the channel measurement lock, no matter
+        what exception was thrown in the process.
+
+        This method prevents exceptions from creating hardware deadlocks.
+        It is the recommended way to isolate channels during measurement.
+        """
+        self._channel_measurement_lock.acquire()
+        try:
+            yield
+        finally:
+            self._channel_measurement_lock.release()
+
+    class Channel(object):
+        """
+        Contains methods for working with a single measurement channel on
+        the level meter. This class is meant to be accessed using the
+        ``channel`` property in `CryomagneticsLM510`
+        """
+
+        def __init__(
```

Comment:
Please use the same parameter names as other instrument channels for consistency